bitwarden_license/src/Commercial.Core/SecretsManager/AuthorizationHandlers/Secrets/SecretAuthorizationHandler.cs

```diff
@@ -109,9 +109,9 @@ public class SecretAuthorizationHandler : AuthorizationHandler<SecretOperationRe
     {
         var (accessClient, userId) = await _accessClientQuery.GetAccessClientAsync(context.User, resource.OrganizationId);
 
-        // All projects should be apart of the same organization
+        // All projects should be in the same organization
         if (resource.Projects != null
-            && resource.Projects.Any()
+            && resource.Projects.Count != 0
```

Comment:
`.Any()` is preferred over `.Count()` the Linq method. 

Since this is an `ICollection` we have access to the property `Count` which is more efficient than `.Any()`.

This has a better breakdown https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1860